### ISSUE TITLE
[Enhancement] Dummy "Receiving on progress, stay tuned" page 

### ISSUE THREAD

**Issue by takagiwa** [stdevPavelmc/FAASGS#15]
Hello,

This is not important.
It will help me if there are dummy index.html that shows "Receiving" before generate index.php.
Maybe copy that just after "mkdir -p ${WBASE}" in sat.sh .

Best regards,

**Comment by stdevPavelmc** [comment]
Every Bit counts and it's important.

You are absolutely right, a "Receiving..." page will tell the user that there is a process going on.

A 1 minute reload trigger will update it until the final page is ready, and that is easy to implement.

Good catch!

**Comment by takagiwa** [comment]
Hello

I think you already did that.
If not,

1. make new html file as "/etc/sat_data/inprogress.html".
2. add "cp "${CONFPATH}/inprogress.html" "${WBASE}/index.html" just after "mkdir -p ${WBASE}"
3. add "rm "${WBASE}/index.html" just before "cp "${CONFPATH}/index.php" "${WBASE}/"

seems worked.


Best regards,

**Comment by stdevPavelmc** [comment]
Yep, that's the intended way, I was crafting a nice looking and mobile friendly loading page but family issues on the weekend make me stop to work on this.

I have a branch open for this locally, no worry, it will came up on the week.
